smack-extensions/src/main/java/org/jivesoftware/smackx/receipts/DeliveryReceipt.java

```diff
@@ -22,14 +22,14 @@
 import org.jivesoftware.smack.packet.ExtensionElement;
 import org.jivesoftware.smack.packet.Message;
 import org.jivesoftware.smack.provider.EmbeddedExtensionProvider;
-import org.jivesoftware.smack.util.StringUtils;
 import org.jivesoftware.smack.util.XmlStringBuilder;
 
 /**
  * Represents a <b>message delivery receipt</b> entry as specified by
  * <a href="http://xmpp.org/extensions/xep-0184.html">Message Delivery Receipts</a>.
  *
  * @author Georg Lukas
+ * @author Paul Schaub
```

Comment:
Please do not take this the wrong way, but could you remove yourself from the javadoc authors list? I'd like to have there only people substantial authored the code. (You could however, add your name to the copryright line).